pkg/webhooks/prometheusrule/prometheusrule.go

```diff
@@ -0,0 +1,188 @@
+package prometheusrule
+
+import (
+	"fmt"
+	"net/http"
+
+	hookconfig "github.com/openshift/managed-cluster-validating-webhooks/pkg/config"
+	"github.com/openshift/managed-cluster-validating-webhooks/pkg/webhooks/utils"
+
+	admissionregv1 "k8s.io/api/admissionregistration/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	admissionctl "sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+const (
+	WebhookName string = "prometheusrule-validation"
+	docString   string = `Managed OpenShift Customers may not create PrometheusRule in namespaces managed by Red Hat .`
```

Comment:
```suggestion
	docString   string = `Managed OpenShift Customers may not create PrometheusRule in namespaces managed by Red Hat.`
```

Unneeded white space.